week3/src/components/Header.jsx

```diff
@@ -0,0 +1,72 @@
+// styled API를 사용하여 스타일링
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import "./SingleCard.css";
+
+const Header = ({ counter, length }) => {
+  const [animate, setAnimate] = useState(false);
+  useEffect(() => {
+    if (counter != 0) {
```

Comment:
```suggestion
    if (counter !== 0) {
```
데이터 타입도 확인하는 엄격한 동등 비교 연산자를 사용하는게 더 안정적이게찌!?

Reply:
헐 마자 나 왜저렇게 썼냐~ 더 신경써야겟다!!